examples/taco/nodejs/src/index.ts

```diff
@@ -46,7 +47,7 @@ const runExample = async () => {
     hasPositiveBalance.requiresSigner(),
     'Condition requires signer',
   );
-  const ritualId = 2; // Replace with your own ritual ID
+  const ritualId = 1; // Replace with your own ritual ID
```

Comment:
Similar comment in python demo job. We should make this modifiable without needing to change the code. Obviously not necessarily for this PR, but something to follow-up on.

Reply:
Agreed, this needs to be an input parameter at some layer.

Reply:
I don't mind we do that, but is that a comment specific to running the example as a GH action, or for the example script in general?